packages/venia-ui/lib/components/CheckoutPage/ItemsReview/itemsReview.css

```diff
@@ -1,12 +1,60 @@
-.text_content {
-    padding: 25px;
-    padding-top: 50px;
+.total_quantity {
     font-weight: bold;
 }
 
-.container {
-    min-height: 20rem;
+.show_all_items_button {
+    composes: root_normalPriority from '../../Button/button.css';
+    width: 100%;
+    border: none;
+}
+
+.show_all_items_button > span {
+    font-weight: bold;
+    font-size: 0.9rem;
+}
+
+.show_all_footer_container {
+    border-top: 1px solid rgb(var(--venia-border));
+    padding: 0.5rem 0rem;
+}
+
+.items_review_container {
+    min-height: 2rem;
     border: 1px solid rgb(var(--venia-border));
     border-radius: 5px;
+}
+
+.items_container {
+    display: grid;
+    gap: 2rem;
     padding: 1.5rem 1rem;
 }
+
+.item_root {
+    display: grid;
+    grid-template-columns: 1fr 4fr;
+    gap: 0.5rem;
+}
+
+.item_thumbnail {
+    grid-column: 1 / span 1;
+    grid-row: 1 / span 3;
```

Comment:
Good syntax; we do prefer these properties and this value structure.

That said, explicitly placing items into the grid is usually a sign of overcomplicating things. I'll check out the branch and make a call in a bit.

Reply:
After review, I think this is a good case for explicit placement. 👍